sammo.py

```diff
@@ -51,7 +51,7 @@ def createSimuGps(self) -> [SammoSimuGpsBtn, ThreadSimuGps]:
         button = SammoSimuGpsBtn(self.iface.mainWindow(), self._toolBar)
         button.onChangeSimuGpsStatusSignal.connect(self.onChangeSimuGpsStatus)
         testFilePath = os.path.join(
-            self.pluginFolder(), "src/core/gps_coordinates_test.fic"
+            self.pluginFolder(), "src/core/trace_simu_gps.csv"
```

Comment:
You should use `os.path.join("src", "core", "trace_simu_gps.csv")` to be cross-platform.